cdbot/constants.py

```diff
@@ -99,6 +99,11 @@ class Exchange:
 QUOTES_BOT_ID = 292953664492929025
 LOGGING_CHANNEL_ID = int(environ.get("LOGGING_CHANNEL_ID", "538494690601992212"))
 WELCOME_BOT_ID = 155149108183695360
+POLL_CHANNEL_ID = int(environ.get("POLL_CHANNEL_ID", "574878887625097227"))
+POLL_WEBHOOK_ID = int(environ.get("POLL_WEBHOOK_ID"))
```

Comment:
a) Your PR should note that this env var needs to be added
b) Because this is a secret, it should use `getenv` instead

```suggestion
POLL_WEBHOOK_ID = int(getenv("POLL_WEBHOOK_ID"))
```

Reply:
a is addressed in the the first comment